js/functional/fibonacci.js

```diff
@@ -16,8 +34,20 @@ function generateFibonacciUsingLoop(sequence) {
  * @param {number} sequence
  */
 function generateFibonacciUsingRecursive(sequence) {
-  // write your code here
-  return [];
+  if (sequence <= 0) {
+    return [];
+  } else if (sequence <= 1) {
+    return [0];
+  } else if (sequence <= 2) {
+    return [0, 1];
+  } else {
+    const currentArray = generateFibonacciUsingRecursive(sequence - 1);
+    const nextNumber =
+      currentArray[currentArray.length - 1] +
+      currentArray[currentArray.length - 2];
+    currentArray.push(nextNumber);
+    return currentArray;
+  }
```

Comment:
You don't need this "else" statement since "return" will exit the entire method and give the following value as its return value.

Reply:
Please check my further review here: https://github.com/ngodingbang/rwid-git/pull/9#discussion_r1297433757.